pkg/config/policies.go

```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
 	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName], policyName, policies)
 }
 
+// AddChannelPolicy modifies an existing channel policy configuration.
+// When the policy exists it will overwrite the existing policy.
+func (c *ConfigTx) AddChannelPolicy(modPolicy, policyName string, policy Policy) error {
+	return addPolicy(c.updated.ChannelGroup, modPolicy, policyName, policy)
+}
+
+// RemoveChannelPolicy removes an existing channel policy configuration.
```

Comment:
Mentioning configuration at the end is a bit wordy. Probably good enough to say `RemoveChannelPolicy removes an existing channel level policy.`

Reply:
done.